pkg/server/objects_operations.go

```diff
@@ -2,38 +2,78 @@ package server
 
 import (
 	"context"
+	"time"
 
 	"github.com/codenotary/immudb/pkg/api/schema"
 	"github.com/codenotary/immudb/pkg/api/schemav2"
 	"github.com/golang/protobuf/ptypes/empty"
 )
 
 func (s *ImmuServer) LoginV2(ctx context.Context, loginReq *schemav2.LoginRequest) (*schemav2.LoginResponseV2, error) {
-	return nil, nil
+	username := []byte(loginReq.Username)
+	password := []byte(loginReq.Password)
+	session, err := s.OpenSession(ctx, &schema.OpenSessionRequest{
+		Username:     username,
+		Password:     password,
+		DatabaseName: loginReq.Database,
+	})
+	if err != nil {
+		return nil, err
+	}
+	expirationTimestamp := int32(0)
+	if s.Options.SessionsOptions.MaxSessionInactivityTime > 0 {
+		expirationTimestamp = int32(time.Now().Add(s.Options.SessionsOptions.MaxSessionInactivityTime).Unix())
```

Comment:
expirationTime is not exactly the same as inactivity time

Reply:
What should we here return in order to inform user explicitly what he should do?
I mean - expirationTimestamp is a hint, that in x minutes he should refresh token / open new session

Reply:
the inactivity time is setting for how long the server should keep the session alive despite the client has not sent any request for a while (flag --session-timeout).
Expiration time refers to the longest age of a session, even if some activity is continuously done (flag: --max-session-age-time duration)

Probably it's worth sending both values

Reply:
Resolved in https://github.com/codenotary/immudb/pull/1546